dev-docs/bidders/visiblemeasures.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: bidder
+title: VisibleMeasures
+description: Prebid VisibleMeasures Bidder Adapter
+biddercode: visiblemeasures
+usp_supported: true
+gdpr_supported: true
+coppa_supported: true
+schain_supported: true
+floors_supported: true
+media_types: banner, video, native
```

Comment:
If you have multi format support, I would strongly recommend to add `multiformat_supported` (will-bid-on-any, will-bid-on-one, will-not-bid) as a meta data field, so publishers and sales houses know how you handle multi format.

`safeframes_ok` is also a very helpful meta information.

Reply:
That data was added